Sources/Html/Attributes.swift

```diff
@@ -293,6 +293,19 @@ public func draggable<T>(_ value: Draggable) -> Attribute<T> {
   return .init("draggable", value == .auto ? nil : value.rawValue)
 }
 
+public enum Enctype: String {
+  case formURLEncoded = "application/x-www-form-urlencoded"
```

Comment:
Just chatted with @mbrandonw about how we do naming in the library and I think our general rule is to naively camelcase everything based on how it appears in HTML. For example, this line would be:

```suggestion
  case applicationXWwwFormUrlencoded = "application/x-www-form-urlencoded"
```